pandas/core/ops/docstrings.py

```diff
@@ -233,6 +233,43 @@ def _make_flex_doc(op_name, typ):
 dtype: float64
 """
 
+_eq_example_SERIES = """
+Examples
+--------
+>>> s1 = pd.Series([1, 2, 3, 4, 5])
+>>> s2 = pd.Series([1, 2, 5, 9, 2])
+>>> s1.eq(s2)
+Out:
```

Comment:
I don't think you need the `Out`.

Reply:
Hey guys
It was supposed to be a quick wet your feet exercise

its been over 8 committees,
it seems that you'r not quite sure how exactly you want me to fill these out..

Please send me a (pre-approved format), by which I can complete the task

Thanks in advance
Ran

Reply:
@RanDan363 you can look at the other docstrings in this module